flowsa/methods/flowbysectormethods/GHG_national_m2_common.yaml

```diff
@@ -60,18 +60,22 @@ attribution_sources:
 _sources_to_cache:
   Detail_Use_YEAR:  ## Update year
     !include:Detail_Use_SUT.yaml
+    industry_spec: !include:USEEIOv2_detail_target.yaml:industry_spec
```

Comment:
Fine by me if you want to drop it - I agree that it won't make a difference for this FBS. I included the industry spec here because if we are pulling in an industry spec from Detail_USE_SUT.yaml, I figured we might as well pull in the target industry spec instead of what we have in the Detail Use yaml. But I don't have strong feelings